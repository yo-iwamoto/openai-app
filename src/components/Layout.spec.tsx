import { Layout } from './Layout';
import { render } from '@testing-library/react';

describe('[component] Layout', () => {
  it('snapshot', () => {
    const { asFragment } = render(
      <Layout>
        <h1>Hello World!</h1>
      </Layout>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
