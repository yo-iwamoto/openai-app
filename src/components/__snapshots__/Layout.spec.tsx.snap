// Vitest Snapshot v1

exports[`[component] Layout > snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Hello World!
    </h1>
  </div>
</DocumentFragment>
`;
